backend/import/toBoosterCard.js

```diff
@@ -4,52 +4,50 @@ const uuidV1 = require("uuid").v1;
 const toBoosterCard = (setCode) => (mtgjsonCard, index, rawCards) => {
   let {
     name,
+    faceName,
     frameEffects,
     number,
     layout,
     colors,
     colorIdentity,
-    names = [],
+    otherFaceIds,
     convertedManaCost,
     types,
     supertypes = [],
     subtypes = [],
     manaCost,
     url,
-    scryfallId,
+    identifiers = [],
     rarity,
     power,
     toughness,
     loyalty,
     text,
     uuid = `dr4ft-${uuidV1()}`
   } = mtgjsonCard;
-
   if (supertypes.includes("Basic")) {
     rarity = "basic";
   }
 
-  if (/split|aftermath/i.test(layout))
-    name = names.join(" // ");
-
   const {isDoubleFaced, flippedCardURL, flippedIsBack, flippedNumber} = getDoubleFacedProps(mtgjsonCard, rawCards);
   const color = upperFirst(getColor(mtgjsonCard, rawCards));
 
   return {
     uuid,
     name,
-    names,
+    faceName,
+    otherFaceIds,
     color,
     colors,
     colorIdentity,
     setCode,
-    scryfallId,
     cmc: convertedManaCost || 0,
     number,
     type: types[types.length - 1],
     manaCost: manaCost || "",
     rarity: upperFirst(rarity),
-    url: url || `https://api.scryfall.com/cards/${scryfallId}?format=image`,
+    url: url || `https://api.scryfall.com/cards/${identifiers.scryfallId}?format=image`,
+    scryfallId: identifiers.scryfallId,
```

Comment:
Any reason you popout scryfallId if we're not gonna use it and/or you can call to identifiers.scryfallId instead?

Reply:
My first approach was to add identifiers and remove scryfallId from the card object. At the end I change the scryfallId value at line 50.

Reply:
Can we look at dropping the outer SFID, since it's redundant and could cause confusion in the future? (would require my 2nd comment to be changed to the x.id.sf too

Reply:
Done